### DISCUSSION TITLE
binary file provided

### DISCUSSION THREAD

**Discussion by xyont** [BriefFiniteElementNet/CompositeSection#2] [General]
hi,

interesting library to try and use, is there any binary file (dll) available on NuGet? provide these file will attract large user.

regards,

**Comment by epsi1on** [comment]
Yes, good idea.
Could you please make a issue with same content here? I'll check issues more often than discussion. Soon it will be there,

**Reply by xyont** [reply]
probably yes, however i need to understand from documentation and tried the library first.
